lib/jsonapi-serializers/serializer.rb

```diff
@@ -316,8 +316,25 @@ def self.serialize(objects, options = {})
       result
     end
 
-    def self.serialize_errors(errors)
-      {'errors' => errors}
+    def self.serialize_errors(raw_errors)
+      {'errors' => errors(raw_errors)}
+    end
+
+    def self.errors(raw_errors)
+      if raw_errors.class == ActiveModel::Errors
```

Comment:
This is actually unsafe in production for non-Rails users because `ActiveModel::Errors` with throw `NameError: uninitialized constant ActiveModel` — this scarily only works in tests because ActiveModel is included as a development dependency.

I usually fix this in one of two ways:

1. Use `if defined?(ActiveModel::Errors) && ...`
1. Don't add the development dependency, and manually construct a stubbed test object that looks enough like ActiveModel::Errors that it works.

I'm fine with 1. in this case because it's nice to be able to use a real ActiveModel in tests.

Whether or not you pick either of those, we also have to decide if we should do explicit type-checking or not — if you do explicit type checking, then you should use `raw_errors.is_a?(ActiveModel::Errors)` instead of `.class ==` to allow subclasses to work. Alternatively, I usually do a duck-typing check like `raw_errors.respond_to?(:full_messages) && raw_errors.respond_to?(:to_hash)` so that it's flexible with any object that _looks_ (quacks) like an ActiveModel::Errors object, but isn't strictly bound to an explicit type. I'm fine with either of these in this case, do whatever you think is best.